phoenix-core/src/test/java/org/apache/phoenix/index/IndexScrutinyMapperTest.java

```diff
@@ -0,0 +1,117 @@
+package org.apache.phoenix.index;
+
+import org.apache.hadoop.hbase.util.Bytes;
+import org.apache.phoenix.mapreduce.index.IndexScrutinyMapper;
+import org.apache.phoenix.query.BaseConnectionlessQueryTest;
+import org.apache.phoenix.schema.PName;
+import org.apache.phoenix.schema.PNameFactory;
+import org.apache.phoenix.schema.PTable;
+import org.apache.phoenix.schema.PTableType;
+import org.apache.phoenix.util.MetaDataUtil;
+import org.apache.phoenix.util.SchemaUtil;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.mockito.Mockito;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+@RunWith(Parameterized.class)
+public class IndexScrutinyMapperTest extends BaseConnectionlessQueryTest {
+    String schema, tableName, viewName, viewIndex, indexName;
+    boolean isNamespaceEnabled;
+    PTable inputTable;
+
+    @Before
+    public void setup() {
+        schema = "S_"+generateUniqueName();
+        tableName = "T_"+generateUniqueName();
+        viewName = "V_"+generateUniqueName();
+        viewIndex = "VI_"+generateUniqueName();
```

Comment:
viewName and viewIndex don't seem to be used in the tests below - probably testGetSourceTableName_index() and testGetSourceTableName_viewIndex() ought to use these instead of tableName. If not, consider removing them.